runner/runner/celery.py

```diff
@@ -1,8 +1,13 @@
+import os
+
 from celery import Celery
 
+REDIS_HOST = os.getenv("REDIS_HOST", "localhost")
+REDIS_PORT = os.getenv("REDIS_PORT", "6379")
+
 app = Celery(
     "runner",
-    broker="redis://redis",
+    broker=f"redis://{REDIS_HOST}:{REDIS_PORT}",
```

Comment:
As mentioned elsewhere, this may change to use RABBITMQ, but this branch already does a lot so that would be a separate effort.